src/components/IndicatorCard/NumberCard/NumberCard.scss

```diff
@@ -0,0 +1,107 @@
+@import '~style/variables';
```

Comment:
`NumberCard` is mostly copy/pasted over from `IndicatorCard` (there should be almost no change to the indicator cards on the city pages).

But a lot of the branching logic is gone now.